### PR TITLE
qemu_monitor: specify vm name when print monitor debug log.

### PR DESCRIPTION
**Pull request by CongLi**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1218964

### CODE REVIEW

**Comment by CongLi** [comment]
22:37:09 DEBUG| (monitor qmpmonitor1-avocado-vt-vm1) Sending command 'cont'

**Comment by CongLi** [comment]
@pingl2016 

Could you please help review it?

Thanks.

File: virttest/qemu_monitor.py [hunk 1]
```diff
@@ -316,8 +316,8 @@ def _log_command(self, cmd, debug=True, extra_str=""):
         :param extra_str: Extra string would be printed in log.
         """
         if self.debug_log or debug:
-            logging.debug("(monitor %s) Sending command '%s' %s",
```
Comment (pingl2016) [inline virttest/qemu_monitor.py]: As the bug described, we had better to add log as belows.   
e.g.
    (virt-tests-vm1.qmpmonitor1)
    (virt-tests-vm1[dst-xxx].qmpmonitor1)
 
So  I think it is better to changed as below.
+            logging.debug("(monitor %s.%s) Sending command '%s' %s",
+                          self.vm.name, self.name, cmd, extra_str)

**Comment by pingl2016** [comment]
@CongLi Shall we need to add VM's site(Src or Dst VM)?

File: virttest/qemu_monitor.py [hunk 2]
```diff
@@ -316,8 +316,8 @@ def _log_command(self, cmd, debug=True, extra_str=""):
...
-                          self.name, cmd, extra_str)
+            logging.debug("(monitor %s-%s) Sending command '%s' %s",
+                          self.name, self.vm.name, cmd, extra_str)
```
Comment (suqinhuang) [inline virttest/qemu_monitor.py]: this fix cause regression before

**Comment by CongLi** [comment]
23:15:50 DEBUG| Send command: {'execute': 'qmp_capabilities', 'id': 'Wwy52CZG'}
23:15:50 INFO | Connecting to monitor '<virttest.qemu_monitor.QMPMonitor> catch_monitor'
23:15:50 DEBUG| (monitor avocado-vt-vm1.catch_monitor) Sending command 'qmp_capabilities'
23:15:50 DEBUG| Send command: {'execute': 'qmp_capabilities', 'id': 'vUVcYZBc'}

**Comment by CongLi** [comment]
@pingl2016 Confirmed with @luckyh , no plan to add dst and src now which need a big update, will do it later.

@pingl2016 @luckyh Could you please help review it again?

Thanks.

**Comment by luckyh** [comment]
I think it should be applied for the response as well.
```
21:30:41 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Sending command 'query-block' 
21:30:41 DEBUG| Send command: {'execute': 'query-block', 'id': '7ndaGV83'}
21:30:41 DEBUG| (monitor qmpmonitor1) Response to 'query-block' (re-formated)
                         ^^^^^^^^^^^
```

**Comment by CongLi** [comment]
03:19:13 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Sending command 'query-cpus'
03:19:13 DEBUG| Send command: {'execute': 'query-cpus', 'id': 'TKPr4ajz'}
03:19:13 DEBUG| (monitor avocado-vt-vm1.qmpmonitor1) Response to 'query-cpus' (re-formated)

@luckyh 
Could you please help review it again?

Thanks.

File: virttest/qemu_monitor.py [hunk 3]
```diff
@@ -710,7 +710,8 @@ def _log_response(self, cmd, resp, debug=True):
         :param debug: Whether to print the commands.
         """
         if self.debug_log or debug:
-            logging.debug("(monitor %s) Response to '%s'", self.name, cmd)
+            logging.debug("(monitor %s.%s) Response to '%s'",
+                          self.vm.name, self.name, cmd)
             for l in resp.splitlines():
                 logging.debug("(monitor %s)    %s", self.name, l)
```
Comment (luckyh) [inline virttest/qemu_monitor.py]: Missing the change for this line?

**Comment by CongLi** [comment]
@luckyh 

10:57:23 ERROR| QMPEventError: QMP event RESET not received after system_reset (monitor 'avocado-vt-vm1.qmpmonitor1')

Updated, could you please help review it again?

Thanks.

**Comment by luckyh** [comment]
Thanks all.

**Pull request merged**